packages/shared/src/lib/boot.ts

```diff
@@ -87,21 +87,32 @@ export async function getBootData(app: string, url?: string): Promise<Boot> {
     params.append('url', url);
   }
 
-  const res = await fetch(`${apiUrl}/boot${appRoute}?${params}`, {
-    method: 'GET',
-    credentials: 'include',
-    headers: { app, 'Content-Type': 'application/json' },
-  });
-  const result = await res.json();
+  try {
+    const res = await fetch(`${apiUrl}/boot${appRoute}?${params}`, {
+      method: 'GET',
+      credentials: 'include',
+      headers: { app, 'Content-Type': 'application/json' },
+    });
 
-  const features = await decrypt(
-    result.exp.f,
-    process.env.NEXT_PUBLIC_EXPERIMENTATION_KEY,
-    'AES-CBC',
-    128,
-  );
+    if (res.status >= 500) {
+      throw new Error('Server error');
```

Comment:
You throw an error inside a try catch and then catch it to throw again. Seems a bit weird

Reply:
The main issue is that new error handling state of react query 😢 
Maybe @capJavert knows how to better avoid it

Reply:
This part is looks ok since fetch won't throw on non 200 status